### PR TITLE
Tina beta launch post

### PR DESCRIPTION
**Pull request by perkinsjr**
Creating a Draft PR for 👀  feel free to edit the content as you see fit. We are also waiting on Scott's Visionary piece. Not this blog post is already 1200 words. We want to keep it under 5 minutes total, on average it would take someone 4 minutes to read in it's current state.

### CODE REVIEW

**Review by DirtyF — APPROVED** [review]

File: content/blog/tina-is-in-beta.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ We also moved and created new navigation menus to better convey the intent of a
 
 ### A New Initialize Command in the CLI
 
-Tina init is my favorite addition to the Tina experience. A single command can bootstrap Tina on a Next.js application and do all the heavy lifting for you. The team spent quite a bit of time working on this, and refining it, to get it just right.  With the command `npx @tinacms/cli init` we will do the following:
+Tina init is my favorite addition to the Tina experience. A single command can bootstrap Tina on a Next.js application and do all the heavy lifting for you. The team spent quite a bit of time working on this, and refining it, to get it just right.The command `npx @tinacms/cli init` ommand currently does the following:
```
Comment (DirtyF) [inline content/blog/tina-is-in-beta.md:67]:
```suggestion
Tina init is my favorite addition to the Tina experience. A single command can bootstrap Tina on a Next.js application and do all the heavy lifting for you. The team spent quite a bit of time working on this, and refining it, to get it just right. The command `npx @tinacms/cli init` command currently does the following:
```

File: content/blog/tina-is-in-beta.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+## Media Manager
+
+Media manager was one of the most important features that we needed for Tina Cloud. Our Cloudinary Media manager allows you to change images, upload new images, and delete ones you no longer need without ever leaving the Tina editing experience. 
+
+I wrote a [blog post announcing it](/blog/manage-your-media-with-cloudinary/) and how to implement it into your application.
+
+## Caching Improvements
+
+Speed and performance have been something we have been actively working on. We introduced some improvements behind the scenes to improve the way we retrieve the data for your site. Tina was already performance first but now it's even better!
```
Comment (DirtyF) [inline content/blog/tina-is-in-beta.md:102]:
```suggestion
Speed and performance have been something we have been actively working on. We introduced some improvements behind the scenes to improve the way we retrieve the data for your site. Tina is carefully built with performance in mind and is now faster! 
```

File: content/blog/tina-is-in-beta.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+## Creating `@tinacms/toolkit`
+
+TinaCMS was built with small modular packages, this meant that we relied heavily on React context. The dependency mismatches from over-modularizing our toolkit,  led to many bugs related to missing context.
+
+Our open-source team created @tinacms/toolkit which incorporates the essentials of Tina all in one place. This simplifies everything for you as a user and Tina as a product.
+
+You can read about all the updates and why decided to make the changes in our pinned [GitHub issue](https://github.com/tinacms/tinacms/issues/1898).
+
+## Vercel Integration
```
Comment (DirtyF) [inline content/blog/tina-is-in-beta.md:112]: Vercel integration is currently available when you add our starter, we're already working on providing a Vercel deploy button for any app added to Tina Cloud.

File: content/blog/tina-is-in-beta.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+We wanted to reduce the friction to almost zero when testing TinaCMS, so we worked on adding Vercel integration. This means if you sign up for an account, you can one-click and deploy in minutes and start playing around with TinaCMS and Tina Cloud, using our Starter. 
+
+## Dashboard Overhaul
+
+When using Tina Cloud in alpha our dashboard UX wasn't a first-class experience and at times could be confusing. We completely overhauled the dashboard, making it easier and quicker to add an application to the Cloud, invite users, and find the important information such as site URL(s) or client-id.
```
Comment (DirtyF) [inline content/blog/tina-is-in-beta.md:118]:
```suggestion
When using Tina Cloud in alpha our dashboard UX wasn't a first-class experience and at times could be confusing. We completely overhauled the dashboard, making it easier and quicker to add an application to the Cloud, invite users, and find the important information such as site URL(s) or Client ID.
```

File: content/blog/tina-is-in-beta.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+If you did use the alpha you will need to sign up again as we made large changes to the way we handle user signup in our backend.
```
Comment (DirtyF) [inline content/blog/tina-is-in-beta.md:120]: link to migration page?

File: content/blog/tina-is-in-beta.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+The Tina Starter was built originally to show "the power of Tina" while it did that, we didn't feel that it showed a real-world example. So we went back to the drawing board and created our new [Tina Starter](/guides/tina-cloud/starter/overview/), which includes a landing page, blog, and about pages. You can edit and rearrange the content and we styled it with TailwindCSS to give it some extra shine! Below is an example of just some of the work you can do:
+
+
+![Quickstart Example](/img/edit-alongside-content.gif)
+
+## Media Manager
+
+Media manager was one of the most important features that we needed for Tina Cloud. Our Cloudinary Media manager allows you to change images, upload new images, and delete ones you no longer need without ever leaving the Tina editing experience. 
+
+I wrote a [blog post announcing it](/blog/manage-your-media-with-cloudinary/) and how to implement it into your application.
```
Comment (logan-anderson) [inline content/blog/tina-is-in-beta.md:100]: Maybe we could also link to the docs?

File: content/blog/tina-is-in-beta.md [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+## Changes to Content Modeling 
+
+Content Modeling is the core of how you interact with TinaCMS and also retrieve your content. We decided to make an important change and be more primitive based in nature. 
+
+This allows for simplistic queries that don't require disambiguation, we believe this will allow you as a developer to craft queries with ease. 
+
+If you used the alpha of Tina you might want to read this article the team put together to explain all of the [changes and how to migrate](/docs/tina-cloud/migration-overview/). 
+
+## Give Us Feedback!
+The whole team is truly excited to enter the beta phase and hope you will check it out and give us honest feedback. We want to hear about your projects or, let us know how Tina Cloud can help your team make progress.
```
Comment (logan-anderson) [inline content/blog/tina-is-in-beta.md:133]: Maybe somewhere in here we should ask folks to join our discord server?

Comment (perkinsjr) [reply content/blog/tina-is-in-beta.md:133]: done

File: content/blog/tina-is-in-beta.md [hunk 8]
```diff
@@ -0,0 +1,137 @@
+---
+title: Tina is in Beta
+date: '2021-07-27T20:00:00-04:00'
```
Comment (logan-anderson) [inline content/blog/tina-is-in-beta.md:3]: Maybe we should change the date to today's date?

File: content/blog/tina-is-in-beta.md [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+author: James Perkins
+last_edited: '2021-07-27T15:39:06.367Z'
```
Comment (logan-anderson) [inline content/blog/tina-is-in-beta.md:5]: Same with last_edited

File: content/blog/tina-is-in-beta.md (hunk 8 above)
Comment (perkinsjr) [reply content/blog/tina-is-in-beta.md:3]: Yup it's on the todo, still editing content

**Review by logan-anderson — APPROVED** [review]

**Pull request merged**